src/pudl_scrapers/bin/ferc_xbrl.py

```diff
@@ -0,0 +1,131 @@
+"""A command line interface (CLI) to archive data from an RSS feed."""
+import argparse
+import json
+import logging
+import re
+import zipfile
+from pathlib import Path
+
+import feedparser
+import requests
+
+import pudl_scrapers.settings
+from pudl_scrapers.helpers import new_output_dir
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+FERC_RSS_LINK = "https://ecollection.ferc.gov/api/rssfeed"
+
+
+def parse_main():
+    """Process base commands from the CLI."""
+    parser = argparse.ArgumentParser(description="Archive filings from RSS feed")
+    parser.add_argument(
+        "-r", "--rss-path", default=FERC_RSS_LINK, help="Specify path to RSS feed"
+    )
+    parser.add_argument(
+        "-y", "--year", default=2021, type=int, help="Specify single year for filter"
+    )
+    parser.add_argument(
+        "-f", "--form-number", default=1, type=int, help="Specify form name for filter"
```

Comment:
It would be good to include the list of known to be valid form numbers in the help message somehow, along with a 1-line description of what data they contain. Otherwise it's not clear what can be provided as an argument here.